### PR TITLE
GH#55397 Updated 4.8 to 4.10 references to 4.10 to 4.12 for EUS updates

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.12+

Issue:
https://github.com/openshift/openshift-docs/issues/55397

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/updating-eus-to-eus-upgrade.adoc [hunk 1]
```diff
@@ -6,17 +6,17 @@
...
 * Review the release notes and product lifecycles for any layered products and OLM Operators. Some may require updates either before or during an EUS-to-EUS update.
-* Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to updating from {product-title} 4.8 to 4.9.
-* Verify that your cluster is running {product-title} version 4.8.14 or later.
-If your cluster is running a version earlier than {product-title} 4.8.14, you must update to a later 4.8.z version before updating to 4.9.
-The update to 4.8.14 or later is necessary to fulfill the minimum version requirements that must be performed without pausing MachineConfigPools.
+* Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.11/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to updating from {product-title} 4.10 to 4.11.
```
Comment (bergerhoffer) [inline modules/updating-eus-to-eus-upgrade.adoc:16]: This link doesn't work - that page didn't exist in 4.11.

However, it does exist for 4.12, so I would update that link and change the text here to say ... 4.11 to 4.12.

File: modules/updating-eus-to-eus-upgrade.adoc [hunk 2]
```diff
@@ -6,17 +6,17 @@
...
+* Verify that your cluster is running {product-title} version 4.10.14 or later.
```
Comment (bergerhoffer) [inline modules/updating-eus-to-eus-upgrade.adoc:17]: This section might no longer be applicable at all  - check w/ SMEs

File: updating/preparing-eus-eus-upgrade.adoc [hunk 3]
```diff
@@ -7,18 +7,18 @@ include::_attributes/common-attributes.adoc[]
...
 * If you encounter issues during or after upgrading to the odd-numbered minor version but before upgrading to the next even-numbered version, then remediation of those issues may require that non-control plane hosts complete the update to the odd-numbered version before moving forward.
 * You can do a partial update by updating the worker or custom pool nodes to accommodate the time it takes for maintenance.
 * You can complete the update process during multiple maintenance windows by pausing at intermediate steps. However, plan to complete the entire update within 60 days. This is critical to ensure that normal cluster automation processes are completed including those associated with certificate rotation.
-* You must be running at least {product-title} 4.8.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.8.z before attempting the EUS-to-EUS update.
+* You must be running at least {product-title} 4.10.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.10.z before attempting the EUS-to-EUS update.
```
Comment (bergerhoffer) [inline updating/preparing-eus-eus-upgrade.adoc:19]: Same comment as above - check if this is still relevant for 4.12

File: updating/preparing-eus-eus-upgrade.adoc [hunk 4]
```diff
@@ -7,18 +7,18 @@ include::_attributes/common-attributes.adoc[]
...
 * Support for RHEL7 workers was removed in {product-title} 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers.
```
Comment (bergerhoffer) [inline updating/preparing-eus-eus-upgrade.adoc:20]: This line might not be applicable anymore either

File: modules/updating-eus-to-eus-upgrade.adoc (hunk 2 above)
Comment (wking) [inline modules/updating-eus-to-eus-upgrade.adoc:17]: We can drop the 4.8.14 mentions entirely.  They had been added to cover [the addition of admin-ack guards][1], ensuring that folks had those in place before moving to 4.9.  But  with 4.10 and later, we have conditional update risk declarations, and there are no supported updates that violate our current build recommendations.  In [the graph-data repo][2]:

```console
$ grep minor_min build-suggestions/4.1[12].yaml
build-suggestions/4.11.yaml:  minor_min: 4.10.16
build-suggestions/4.12.yaml:  minor_min: 4.11.11
$ hack/show-edges.py --cincinnati https://api.openshift.com/api/upgrades_info/graph stable-4.11 | grep '^4[.]10[.]\([0-9]\|1[0-5]\) -.* 4[.]11[.]'
...no hits...
$ hack/show-edges.py --cincinnati https://api.openshift.com/api/upgrades_info/graph fast-4.12 | grep '^4[.]11[.]\([0-9]\|10\) -.* 4[.]12[.]'
...no hits...
```

And if we discover update risks where dangling compute nodes or something might endanger an update, we would address that by declaring conditional update risks via api.openshift.com, and not by hoping that customers find and evaluate update risks declared only in these docs.

[1]: https://github.com/openshift/cincinnati-graph-data/commit/2b20f927ccd7c90718a070170c64d2f61dc72ac1
[2]: https://github.com/openshift/cincinnati-graph-data/tree/2fbdc578fafb06c732fe1768f3a6ea8fbfe8eec7

File: modules/updating-eus-to-eus-upgrade.adoc (hunk 1 above)
Comment (obrown1205) [reply modules/updating-eus-to-eus-upgrade.adoc:16]: Good catch :) done!

File: modules/updating-eus-to-eus-upgrade.adoc (hunk 2 above)
Comment (obrown1205) [reply modules/updating-eus-to-eus-upgrade.adoc:17]: @petr-muller @wking what do y'all think? :)

File: updating/preparing-eus-eus-upgrade.adoc (hunk 3 above)
Comment (obrown1205) [reply updating/preparing-eus-eus-upgrade.adoc:19]: @petr-muller @wking what do y'all think? :)

File: updating/preparing-eus-eus-upgrade.adoc (hunk 4 above)
Comment (obrown1205) [reply updating/preparing-eus-eus-upgrade.adoc:20]: @petr-muller @wking what do y'all think?

Comment (obrown1205) [reply updating/preparing-eus-eus-upgrade.adoc:19]: removed per @wking 's comment :)

File: updating/preparing-eus-eus-upgrade.adoc (hunk 3 above)
Comment (obrown1205) [reply updating/preparing-eus-eus-upgrade.adoc]: removed per @wking 's comment :)

File: modules/updating-eus-to-eus-upgrade.adoc (hunk 2 above)
Comment (obrown1205) [reply modules/updating-eus-to-eus-upgrade.adoc]: removed per @wking 's comment :)

File: updating/preparing-eus-eus-upgrade.adoc [hunk 5]
```diff
@@ -7,18 +7,20 @@ include::_attributes/common-attributes.adoc[]
...
 * You can complete the update process during multiple maintenance windows by pausing at intermediate steps. However, plan to complete the entire update within 60 days. This is critical to ensure that normal cluster automation processes are completed including those associated with certificate rotation.
-* You must be running at least {product-title} 4.8.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.8.z before attempting the EUS-to-EUS update.
-* Support for RHEL7 workers was removed in {product-title} 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers.
-* Node components are not updated to {product-title} 4.9. Do not expect all features and bugs fixed in {product-title} 4.9 to be made available until you complete the update to {product-title} 4.10 and enable all MachineConfigPools to update.
+* Support for RHEL7 workers was removed in OpenShift Container Platform 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers.
```
Comment (skrthomas) [inline updating/preparing-eus-eus-upgrade.adoc:19]:
```suggestion
* Support for RHEL7 workers was removed in {product-title} 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers. 
```

Comment (skrthomas) [reply updating/preparing-eus-eus-upgrade.adoc:19]: Make sure there's no "OpenShift Container Platform" spelled out. Its always {product-title}. Applies also below.

**Review by skrthomas — COMMENTED** [review]
Small thingy, otherwise LGTM :) Nice work!

File: updating/preparing-eus-eus-upgrade.adoc (hunk 5 above)
Comment (obrown1205) [reply updating/preparing-eus-eus-upgrade.adoc:19]: thanks so much! will update :)

**Comment by jboxman-rh** [comment]
@obrown1205, this needs a merge review or not? It has both labels. Thanks.

**Comment by skrthomas** [comment]
@jboxman-rh I removed the peer-review-in-progress label, my b. Peer review is in fact done.

**Comment by jboxman-rh** [comment]
@bergerhoffer looks like this is borked somehow. afaik this should never happen?

**Comment by bergerhoffer** [comment]
@jboxman-rh you're right, CPs to 4.13 shouldn't fail, thanks for pinging me! Let me see what's going on...

**Comment by bergerhoffer** [comment]
Yep, there is a PR (#43712) that I had marked as having an issue last year (went into main but not 4.11). I'm not positive if any of those changes need to go into 4.11+, but as they are now, it can't get cherry-picked without changes. 

I'm going to back out those changes from main so that this PR should pick cleanly. Then we can re-evaluate if anything needs to go forward. Since it's so late in the day, we'll take care of this in the morning. Hang tight til then!

**Comment by bergerhoffer** [comment]
I ended up cherry picking the changes to 4.11+ (instead of backing out from main) since it looked like they were actually appropriate for 4.11+. The CPs for this PR should pick cleanly now. I've issued the CPs and will merge them once they're ready!

**Pull request merged**